src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Badge, Dropdown, Menu, Space, Typography } from "antd";
+import { IconBell } from "../../../icons/Icons";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { LinkButton } from "../../../Buttons/LinkButton";
+import { TYPES, useAnnouncements } from "./announcements-context";
+import { fromNow } from "../../../../utilities/date-utilities";
+import { BORDERED_LIGHT } from "../../../../styles/borders";
+import { PriorityFlag } from "../../../../pages/announcement/components/PriorityFlag";
+import "./announcements.css";
```

Comment:
Instead of using an external style sheet use the styled-components here.  This has 2 benefits for you:

1.  You can use the `theme` variable from `theme-utilities` allowing you to set the color if it is dark or light.
2.  It will automatically scope the css variable to this component.  Otherwise your color of your icon will be used anywhere on this page causing undo cascade issue.

I would wrap the span in the style and then add the padding there instead of inline on the element

Reply:
Please see a7b317100141ed8363d6b9f52e4c481c64b455c1.